domains/Settings/SettingsHome/AccountSettings/FindRecords/index.js

```diff
@@ -0,0 +1,210 @@
+      <View style={styles.horizontalLinePrimary} />
+      {inputs.length > 0 && inputs.map((result) => (
+        <View key={result.key}>
+          <Text style={styles.text}>{result.label}</Text>
+          <View>
+            {edit !== result.key && (
+              <View style={styles.textContainer}>
+                <Text style={styles.text}>{currentData[result.key]}</Text>
+                {result.edit === true && (
+                  <Button
+                    style={{ marginLeft: 'auto' }}
+                    onPress={() => {
+                      setEdit(result.key);
+                    }}
+                  >
+                    Edit
+                  </Button>
+                )}
+              </View>
+            )}
+            {edit === result.key && (
+              <View style={styles.textContainer}>
+                <TextInput
+                  style={{ flex: 3 }}
+                  placeholder={result.value}
+                  mode="outlined"
+                  onChangeText={(text) => updateCurrentData(result.key, text)}
+                />
+                <View style={styles.buttonContainer}>
+                  <IconButton
+                    icon="check"
+                    size={25}
+                    color={theme.colors.primary}
+                    style={styles.svg}
+                    onPress={() => {
+                      setEdit('');
+                    }}
+                  />
+                  <IconButton
+                    icon="window-close"
+                    size={25}
+                    color={theme.colors.primary}
+                    style={styles.svg}
+                    onPress={() => {
+                      setEdit('');
+                    }}
+                  />
+                </View>
+              </View>
+            )}
+          </View>
+          <View style={styles.horizontalLineGray} />
+
+        </View>
+      ))}
+      {submitting ? (
+        <ActivityIndicator
+          size="large"
+          color={theme.colors.primary}
+        />
+      ) : (
+        <Button onPress={() => updateUser()}>Submit</Button>
+      )}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  mainContainer: {
+    paddingLeft: '5%',
+    paddingRight: '5%'
+  },
+  textContainer: {
+    flexDirection: 'row'
+  },
+  buttonContainer: {
+    flexDirection: 'row',
+    marginLeft: 'auto',
+    marginTop: 'auto',
+    marginBottom: 'auto',
+    flex: 1
+  },
+  svg: {
+    marginLeft: 'auto',
+    marginTop: -3,
+    marginBottom: -5
+  },
+  text: {
+    flexShrink: 1,
+    // fontWeight: 'bold',
+    fontSize: 16,
+    color: '#555',
+    marginVertical: 7,
+  },
+  horizontalLineGray: {
+    borderBottomColor: '#D0D0D0',
+    borderBottomWidth: 1,
+    marginTop: 10,
+    marginBottom: 10,
+  },
+  horizontalLinePrimary: {
+    borderBottomColor: theme.colors.primary,
+    borderBottomWidth: 1,
+    marginTop: 10,
+    marginBottom: 10,
+  }
+});
```

Comment:
let's move this to an `index.style.js` file

Reply:
done